cloudbuild.yaml

```diff
@@ -0,0 +1,100 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This is a config file used by Cloud Build and the Cloud Build local runner to 
+# run a CI pipeline to check if any changes done pass tests.
+
+
+steps:
+
+- name: 'docker/compose'
+  id: 'Launch Webserver and HAPI Server'
+  args: ['-f', './docker/openmrs-compose.yaml', '-f', './docker/sink-compose.yml',  'up', '--force-recreate', '--remove-orphan', '-d']
+
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'Wait for Servers Start'
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - e2e-tests/wait_for_start.sh --use_docker_network
+
+- name: 'adoptopenjdk/maven-openjdk8'
+  id: 'Compile Bunsen and Pipeline'
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - mvn --no-transfer-progress -e -f bunsen install; mvn --no-transfer-progress -e -f pipelines install
+  waitFor: ['-'] 
+
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'Build Batch Pipeline Image'
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - cd pipelines/batch;  docker build -t gcr.io/${PROJECT_ID}/batch-pipeline:${_TAG} .
```

Comment:
Now that you have removed `_PROJECT_ID`, how does `PROJECT_ID` being set when people run e2e in local mode? Are we still pushing docker images to the Container Registry when building locally (we should not)?

Reply:
Using the local Cloud Build tool requires the Cloud SDK to be installed. `PROJECT_ID` will be replaced with the project id that the SDK was initialized with. 

For `_TAG`, I added comments about how to substitute it